_includes/head.html

```diff
@@ -6,14 +6,14 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <title>Open Source Design</title>
+    <title>Open Source Design events</title>
```

Comment:
:D @simonv3 

@evalica I was thinking because »Open Source Design« is the organization name hence capitalized, and »events« is the section of the site.